src/librbd/ExclusiveLock.cc

```diff
@@ -254,6 +254,40 @@ void ExclusiveLock<I>::handle_peer_notification(int r) {
 }
 
 template <typename I>
+int ExclusiveLock<I>::assert_header_locked() {
+  CephContext *cct = m_image_ctx.cct;
+  ldout(cct, 10) << dendl;
```

Comment:
@dillaman It needs `<< this << " " << __func__ << ` prefix to follow this file convention (the same is for other ldout/lderr in this function).